Model/ProcessedOrdersRegistry.php

```diff
@@ -0,0 +1,40 @@
+<?php
+declare(strict_types=1);
+
+namespace MageOS\CommonAsyncEvents\Model;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Sales\Api\Data\OrderInterface;
+
+/**
+ * Save a list of already processed orders during this request.
+ */
+class ProcessedOrdersRegistry
+{
+    const FIELD_NAME_IDENTIFIER = 'entity_id';
```

Comment:
Is this being used or could this be cleaned up?

Reply:
You're right, can be cleaned up.